components/datetime/src/mock/time_zone.rs

```diff
@@ -36,32 +40,75 @@ use core::str::FromStr;
 #[allow(clippy::exhaustive_structs)] // this type will not add fields (it is largely an example type)
 pub struct MockTimeZone {
     /// The GMT offset in seconds.
-    pub gmt_offset: GmtOffset,
+    pub gmt_offset: Option<GmtOffset>,
     /// The IANA time-zone identifier
     pub time_zone_id: Option<TimeZoneBcp47Id>,
     /// The CLDR metazone identifier
     pub metazone_id: Option<MetaZoneId>,
     /// The time variant e.g. "daylight" or "standard"
     pub time_variant: Option<TinyStr8>,
+    /// The fallback of GMT offset if it is set to None.
+    pub gmt_offset_fallback: String,
```

Comment:
Note: The intent was that this becomes a field of `TimeZoneFormatsV1`, not a field of `MockTimeZone`

Reply:
Heh, and I see that this is exactly what you are doing in https://github.com/unicode-org/icu4x/pull/2253 :laughing: